AucTeX versions up to and including 10.0g have a bug in the
calculation of the offsets for the start of a region.  This affects
C-c C-r commands where the start of the region does not lie on the
start of a line.  It also affects regeneration of single previews.  To
correct this, apply the following patch.

*** tex-buf.el.~1.163.~	Mon Oct  1 15:40:31 2001
--- tex-buf.el	Mon Oct  1 15:45:27 2001
***************
*** 123,130 ****
      (TeX-region-create (TeX-region-file TeX-default-extension)
  		       (buffer-substring begin end)
  		       (file-name-nondirectory (buffer-file-name))
! 		       (count-lines (save-restriction (widen) (point-min))
! 				    begin)))
    (TeX-command (TeX-command-query (TeX-region-file)) 'TeX-region-file))
  
  (defun TeX-command-buffer ()
--- 123,131 ----
      (TeX-region-create (TeX-region-file TeX-default-extension)
  		       (buffer-substring begin end)
  		       (file-name-nondirectory (buffer-file-name))
! 		       (+ (count-lines (save-restriction (widen) (point-min))
! 				    begin)
! 			  (save-excursion (goto-char begin) (if (bolp) 0 -1)))))
    (TeX-command (TeX-command-query (TeX-region-file)) 'TeX-region-file))
  
  (defun TeX-command-buffer ()
